.github/workflows/njsscan-analysis.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow integrates njsscan with GitHub's Code Scanning feature
```

Comment:
This crypto ticker is NOT a nodejs app (it's browser-based / client-side javascript), please remove njsscan from this pull request. Sorry, thanks for your effort though! :)